update.sh

```diff
@@ -3,6 +3,16 @@
 KERNEL_VERSION_RPI1=3.18.0-trunk-rpi
 KERNEL_VERSION_RPI2=3.18.0-trunk-rpi2
 
+RASPBIAN_ARCHIVE_KEY_DIRECTORY="https://archive.raspbian.org"
+RASPBIAN_ARCHIVE_KEY_FILE_NAME="raspbian.public.key"
+RASPBIAN_ARCHIVE_KEY_URL="${RASPBIAN_ARCHIVE_KEY_DIRECTORY}/${RASPBIAN_ARCHIVE_KEY_FILE_NAME}"
```

Comment:
I knew I needed `raspbian.public.key` as a variable/constant too and in order to _not repeat that string_ I did it they way as it's implemented (and I've used such a construct a lot in the past).

Your solution would work too. 
Mine has the slight advantage that you don't have to think about the value of the `KEY_FILE_NAME` constant/variable.